src/ctap/mod.rs

```diff
@@ -848,13 +850,19 @@ where
                     CtapState::<R, CheckUserPresence>::PIN_PROTOCOL_VERSION,
                 ]),
                 max_credential_count_in_list: MAX_CREDENTIAL_COUNT_IN_LIST.map(|c| c as u64),
-                // #TODO(106) update with version 2.1 of HMAC-secret
+                // TODO(#106) update with version 2.1 of HMAC-secret
                 max_credential_id_length: Some(CREDENTIAL_ID_SIZE as u64),
                 transports: Some(vec![AuthenticatorTransport::Usb]),
                 algorithms: Some(vec![ES256_CRED_PARAM]),
                 default_cred_protect: DEFAULT_CRED_PROTECT,
                 min_pin_length: self.persistent_store.min_pin_length()?,
                 firmware_version: None,
+                max_cred_blob_length: None,
+                // TODO(kaczmarczyck) update when extension is implemented
+                max_rp_ids_for_set_min_pin_length: None,
+                remaining_discoverable_credentials: Some(
```

Comment:
Just making sure this is alright with information leakage here. On one side we only expose some credentials according to some permissions and here we expose the count (including those that are not "visible") according to the same permissions?

Reply:
The general idea is to protect the relationship to a specific relying party. It should be impossible to somehow call `has_credential(rp_id)`. Your attack on `GetInfo` only let's you check if a credential was created, not for whom. If you are in a position to guess that relying party, this extra bit probably doesn't help much.

Reply:
I see, all good then.